python/src/deltachat/events.py

```diff
@@ -48,6 +59,10 @@ def ac_log_line(self, message):
         if self.logid:
             locname += "-" + self.logid
         s = "{:2.2f} [{}] {}".format(elapsed, locname, message)
+        if message.startswith("DC_EVENT_WARNING"):
+            s = bcolors.WARN + s + bcolors.ENDC
+        if message.startswith("DC_EVENT_ERROR"):
+            s = bcolors.ERROR + s + bcolors.ENDC
```

Comment:
cool, but is this related to the pr?

Reply:
Yes, I would have saved lots of time if I had not overseen this warning :wink:
I thought it's a rather small change but if you say it's better I'll make another pr.

Reply:
I am also not sure how well this works on windows ... i think in pythonh land one usually uses the https://pypi.org/project/colorama/  package.  You can also keep it as is but make it if-conditional on the platform.